### PR TITLE
Changes in styling of components, remove unnecessary lines of code, adjust reponsivity of the Header

### PR DESCRIPTION
**Pull request by gp231982**
1. Removing underlying the Links, NavLinks of Header, and of every mapped PersonTile nad MovieTile.
2. Resizing of BlankPicture.png in MovieDetails Component to have the same size as the API image of Crew and Cast.
3. Modifying of Header to have better responsiveness for different devices:
4. Modify the theme: add more breakpoints/colors and use them in styled.js for the Header.
5. Remove unnecessary files like pictures, not-used code, and console.logs.
6. Changing the Link name for HomeLink in index.js of the Header component.
7. Adjusting imports when necessary.

### CODE REVIEW

**Review by KamilJustynski — APPROVED** [review]
Its okej, good job :)

**Review by GracjanaKromska — APPROVED** [review]
It looks good. I approve

File: src/slices/movieDetailSlice.js
Comment (KamilJustynski) [inline src/slices/movieDetailSlice.js:1]: This change is unnecessary and inconsistent with the state of affairs

**Review by KamilJustynski — APPROVED** [review]
Good job!

**Pull request merged**